addons/mail/static/src/models/partner/partner.js

```diff
@@ -232,6 +232,12 @@ function factory(dependencies) {
             default: "",
         }),
         email: attr(),
+        /**
+         * This field containt the "mail" format :
+         * name <email@domain>.
+         * @see base/models/res_partner.py
+         */
+        email_formatted: attr(),
```

Comment:
Looks unused? Would not introduce this field if not necessary. And as I said in a previous review, this looks more like a computed fields.